OpenLaTeXStudio/EditorModule/src/latexstudio/editor/LaTeXSettingsPanel.java

```diff
@@ -198,6 +193,11 @@ public void run() {
 
     }
     
+    @SettingListener( setting = ApplicationSettings.Setting.AUTOCOMPLETE_ENABLED )

```

Comment:
This made me wonder.. if I wanted to fire the same property change event from two different classes, wouldn't I fall into infinite loop, since we have the same annotations for writers and listeners?